web/src/main/java/pl/isa/alphateam/service/CustomerService.java

```diff
@@ -0,0 +1,63 @@
+package pl.isa.alphateam.service;
+
+
+import pl.isa.alphateam.customer.Customer;
+import pl.isa.alphateam.customer.CustomerDto;
+import pl.isa.alphateam.customer.CustomerMapper;
+import pl.isa.alphateam.customer.Login;
+import pl.isa.alphateam.parser.JSONParserCustomer;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static pl.isa.alphateam.parser.JSONParserCustomer.saveCustomerInDatabase;
+
+
+public class CustomerService {
+
+    public static void createNewCustomerRecord(CustomerDto customerDto) {
+        Customer customer = CustomerMapper.mapToCustomer(customerDto);
+        List<Customer> customerList = new ArrayList<>();
+        try {
+            customerList = JSONParserCustomer.getCustomers1();
+
+        } catch (IOException e) {
+            System.out.println("We have problem with database");
```

Comment:
Wypisanie do konsoli to trochę za mało, na przyszłość przydałoby się przekazać jakąś wartość z powrotem do kontrolera, żeby móc użytkownika wyświetlić informację, że czegoś się nie udało zrobić